plugins/osdn/ovs/controller.go

```diff
@@ -142,6 +142,24 @@ func (c *FlowController) Setup(localSubnetCIDR, clusterNetworkCIDR, servicesNetw
 		glog.V(5).Infof("[SDN setup] docker setup success:\n%s", out)
 	}
 
+	// Disable iptables for linux bridges (and in particular lbr0), ignoring errors.
+	// (This has to have been performed in advance for docker-in-docker deployments,
+	// since this will fail there).
+	_, _ = exec.Command("modprobe", "br_netfilter").CombinedOutput()
+
+	// Hack around the Kube proxier also sets bridge-nf-call-iptables after we do
+	go func() {
+		for i := 0; i < 20; i++ {
+			if val, err := sysctl.GetSysctl("net/bridge/bridge-nf-call-iptables"); val != 0 || err != nil {
+				glog.V(5).Infof("Setting bridge-nf-call-iptables=0 (currently %d)", val)
+				if err := sysctl.SetSysctl("net/bridge/bridge-nf-call-iptables", 0); err != nil {
+					glog.Warningf("Could not set net.bridge.bridge-nf-call-iptables sysctl: %s", err)
+				}
+			}
+			time.Sleep(time.Second)
```

Comment:
I wouldn't consider this a real fix. If we need hooks for pre/post setup notifications or blocking, we should plumb those.